ajax/libs/jquery.gray/package.json

```diff
@@ -4,16 +4,31 @@
   "version": "1.4.4",
   "description": "Make an image gray in all browsers",
   "license": "MIT",
-  "author": "kununu",
+  "author": "Karl Horky <karl.horky@gmail.com>",
   "keywords": [
     "gray",
     "grey",
     "grayscale",
     "images",
-    "image"
+    "image",
+    "jquery",
+    "jquery-plugin"
   ],
   "repository": {
     "type": "git",
     "url": "git://github.com/karlhorky/gray.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/karlhorky/gray.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/**/*",
```

Comment:
1. Please make sure if we need `main.css`. It seems to be used only in demo page.
2. We don't  need `normalize.css` in this repo because it existed on CDNJS https://cdnjs.com/libraries/normalize
Thanks.